### DISCUSSION TITLE
effects_UNSTABLE  vs effects

### DISCUSSION THREAD

**Discussion by lauhon** [facebookexperimental/Recoil#1925] [Q&A]
Hi,

when creating an atom we have the possibility to add `effects`. For some things it is recommended to not use `effects` but `effects_UNSTABLE`.

What is the difference between the two, the term `_UNSTABLE` makes me wonder if I should use it in production?

**Comment by Cpt-Falcon** [comment]
Bump great question I would also like to know the answer

**Comment by drarmstr** [comment]
Atom effects were originally introduced as `effects_UNSTABLE` to indicate that the API was still new and evolving.  It has since matured and now exposed as just `effects`.  All documentation referring to `effects_UNSTABLE` should be updated to `effects`.  Do you have a link of something that should be updated?

**Reply by Cpt-Falcon** [reply]
You can still use effects_UNSTABLE on the actual atom itself as part of recoil 7.4.0. So we were wondering if there was any difference or if this was just for backwards compatibility

**Reply by drarmstr** [reply]
It still accepts `effects_UNSTABLE` just for backward compatibility to avoid breaking existing code.  They are equivalent.
